src/main/java/pages/PaymentPage.java

```diff
@@ -0,0 +1,48 @@
+package pages;
+
+import common.AbstractComponent;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class PaymentPage extends AbstractComponent {
+//test
+    public PaymentPage(WebDriver driver)  { super(driver); }
+
+    //WebElement cardNumber = driver.findElement(By.xpath("//div[@data-card-fields='number']"));
```

Comment:
remove unnecessary comments and  apply it to  all page

Reply:
Actually, I can't reach Credit Card Element that why I couldn't do anything. I make an inside comment line for all Credi Cart Elements.